app/stylesheet/settings.scss

```diff
@@ -16,7 +16,7 @@
 }
 
 div:has(.bx--checkbox-wrapper){
-  margin-bottom: 2rem;
+  margin-bottom: 0.2rem;
```

Comment:
This will break the alignment of my other forms. Can you write it like this?

```
.evacuate_form, .schedule_form, .live_migrate_form{
  div:has(.bx--checkbox-wrapper){
    margin-bottom: 2rem;
  }
}
```

Reply:
in addition to the :

`div:has(.bx--checkbox-wrapper){
   margin-bottom: 2rem;
}` 

?
to write your css code after this one?

Reply:
No, replace it.